website/docs/src/scss/_code.scss

```diff
@@ -0,0 +1,118 @@
+/* CODE */
+
+.codeBlockContainer__-_-node_modules-\@docusaurus-theme-classic-lib-next-theme-CodeBlock-styles-module {
+  border: solid 1px var(--ifm-toc-border-color);
+}
+
+code {
+  border: none !important;
+  background: transparent !important;
+}
+
+.navbar {
```

Comment:
I don't think this belongs to the `code` scope, consider moving it to another file